www/_posts/2018-01-29-plugins-release.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+author:
+    name: Suraj Pindoria
+    url: https://twitter.com/surajpindoria
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin-camera@4.0.2
+* cordova-plugin-file-transfer@1.7.1
+* cordova-plugin-inappbrowser@2.0.2
+* cordova-plugin-media@5.0.2
+* cordova-plugin-splashscreen@5.0.2
+
+Our last plugin release had an issue which made it impossible to install from npm. These point releases of the plugins fix the bug allowing you to install from npm as well as git url.
```

Comment:
@surajpindoria you can get rid of this statement as it was done in the last plugin release on Dec 30th.

Reply:
Oops, I definitely did not copy/paste from the last blog post 😜